.gitignore

```diff
@@ -23,4 +23,6 @@ yarn-debug.log*
 yarn-error.log*
 
 .eslintcache
-.idea/prettier.xml
\ No newline at end of file
+.idea/prettier.xml
+.idea/misc.xml
+.idea/codeStyles/Project.xml
```

Comment:
I think it would be better to add the whole .idea folder? I am also going to delete the .idea folder.

Reply:
alright

Reply:
Sorry was not clear earlier. I meant including the whole .idea folder in .gitignore. Maybe u could also delete the .idea from our repo?